src/services/auditLogService.ts

```diff
@@ -0,0 +1,105 @@
+import * as gqlTypes from '../typeDefs/gqlTypes.js'
+// import * as dbSchema from '../typeDefs/dbSchema.js'
+import { ErrorCode, Result } from '../result.js'
+import { dbInstance } from '../firebaseDb.js'
+import { logger } from '../logger.js'
+
+export async function getAuditLogById(id: string): Promise<Result<gqlTypes.AuditLogs>> {
```

Comment:
Realistically I think we will want to query them by latest or by entity type. 
Ex facilities changes in the last month


We might want a get by id later for an undo functionality though but probably not important at this moment